app/src/main/java/com/fstyle/androidtrainning/screen/main/playingfragments/lyrics/LyricsFragment.java

```diff
@@ -1,23 +1,52 @@
 package com.fstyle.androidtrainning.screen.main.playingfragments.lyrics;
 
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
 import android.os.Bundle;
+import android.os.Environment;
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports